src/helpers/agent.ts

```diff
@@ -44,6 +44,11 @@ export function setupAgent(options: {
   networks: [IndyVdrPoolConfig, ...IndyVdrPoolConfig[]]
   logger?: Logger
 }) {
+  indyVdr.setLedgerTxnCache({
```

Comment:
Wondering if this should go in `getIndyVdrProxyAgentModules`? It seems like that's where `indyVdr` gets used so it may make more sense to do setup there? Seems like `setupAgent` is all about Agent related config.

Reply:
I'll move it there. It works either way in the meantime